ros/api/v1/hosts.py

```diff
@@ -206,10 +186,10 @@ def build_system_filters():
         if operating_systems := request.args.getlist('os'):
             modified_operating_systems = []
             for os in operating_systems:
-                os = os.split(" ")[1]
-                if os not in OS_VERSIONS.keys():
-                    abort(400, message='Not a valid RHEL version')
-                modified_operating_systems.append(OS_VERSIONS[os])
+                os_object = {"name": "RHEL"}
+                os_object["major"] = int(os.split(".")[0])
+                os_object["minor"] = int(os.split(".")[1])
```

Comment:
By any chance either from UI or direct api call we will allow the call only for major version?
~~~
GET /api/ros/v1/systems?os=8
~~~

In this case we will get `IndexError: list index out of range` so asking?

Reply:
Yes, we do. Here, in my logic `8.0` will work. I'll fix it.